### DISCUSSION TITLE
Game with polemic content [Solved]

### DISCUSSION THREAD

**Discussion by nefter** [community/community#23993] [New to GitHub]
<p>Can you please help me report this game? It’s offensive, and promotes violence against women. Thanks.</p>
<p>Preview: <a href="https://www.tiktok.com/@esimio.impune/video/6937450711022161157" class="inline-onebox" rel="noopener nofollow ugc">TikTok</a><br>
Repo: <a href="https://github.com/JorgeGit101/femicaos-game" rel="noopener nofollow ugc">https://github.com/JorgeGit101/femicaos-game</a></p>
<p>I know this can be off-topic, but I think is an important issue to address for the community.</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hey <a class="mention" href="/nefter">@nefter</a>  and welcome to the GitHub Community forum!</p>
<p>At the moment the repository seems to be either made private or removed. If you believe the repository violates/violated the Community Guidelines, you can report the user who created the repository. Read more about reporting abuse and spam here:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/building-a-strong-community/reporting-abuse-or-spam#reporting-a-user" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/building-a-strong-community/reporting-abuse-or-spam#reporting-a-user" target="_blank" rel="noopener nofollow ugc">Reporting abuse or spam - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/building-a-strong-community/reporting-abuse-or-spam#reporting-a-user" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/github/building-a-strong-community/reporting-abuse-or-spam</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by nefter** [comment]
<p>Thanks for the support, it worked. Kind regards!</p>
